Src/Test.UnitTests.Sarif.PatternMatcher.Function/HttpAnalyzeFunctionTests.cs

```diff
@@ -51,7 +51,7 @@ public async Task Function_HttpAnalyze_FileWithPAT_Should_Return_SarifLog()
                 log: logger,
                 context: TestHelper.ContextSetup());
 
-            ValidateResult(string.Empty, result, runCount: 1, resultCount: 2, FailureLevel.Note, ignoreRegionContent: true);
+            ValidateResult(string.Empty, result, runCount: 1, resultCount: 1, FailureLevel.Note, ignoreRegionContent: true);
```

Comment:
>ValidateResult [](http://example.com/codeflow?start=12&length=14)

Why did this test result change?

Reply:
Before this change, AdoPat validator didn't implemented the ValidatorBase generating duplicated results.
With my current change, we will remove duplicates